boards/arm/rpi_pico/doc/index.rst

```diff
@@ -151,7 +157,9 @@ UF2 file should be drag-and-dropped to the device, which will flash the Pico.
 Debugging
 =========
 
-The SWD interface can also be used to debug the board. To achieve this,
+The SWD interface can also be used to debug the board. To achieve this, you can
+use a SEGGER JLink debug probe and follow the instruction in
```

Comment:
Debugging section also mentions about J-Link using.
https://github.com/zephyrproject-rtos/zephyr/blame/52fe06495dc2d9bfa394353778c3f211441a65fb/boards/arm/rpi_pico/doc/index.rst#L172-L190

We can replace this lines with
```
west debug --runner jlink
```